lib/coverband/collectors/base.rb

```diff
@@ -76,10 +77,22 @@ def stop_coverage
         raise 'abstract'
       end
 
-      def report_coverage
+      def report_coverage_potentially_forked
```

Comment:
To avoid the extra "abstract" method, can we pull the forking logic up to the base class?

Reply:
Ok. I think i misread this initially. Apologize for that.